app/views/custom_fields/_form.html.erb

```diff
@@ -205,7 +205,13 @@ See COPYRIGHT and LICENSE files for more details.
     <div class="form--field">
       <%= f.check_box :is_required %>
       <div class="form--field-instructions">
-        <p><%= t("custom_fields.instructions.is_required_for_project") %></p>
+        <p><%= t("custom_fields.instructions.is_required") %></p>
```

Comment:
Same as with calculated values I don't see the reason for allowing boolean custom fields to be marked as required. To the user, the fact that the value is not filled out has the same meaning as having unset the value. In the eyes of the user, both would count as "the field has the correct (false) value". 

The problem with this additional requirement is that the other customizables don't treat booleans in that fashion. 

It might therefore be better to change it consistently for all customizables in a separate PR.